src/main/config/application.yml.ctmpl

```diff
@@ -5,7 +5,10 @@ hibernate:
     queries: false
     use_second_level_cache: false
     use_query_cache: false
-
+grails.plugin.databasemigration.changelogLocation: grails-app/migrations
+grails.plugin.databasemigration.changelogFileName: changelog-master.xml
+grails.plugin.databasemigration.updateOnStart: true
+grails.plugin.databasemigration.updateOnStartFileName: changelog-master.xml
```

Comment:
This is yaml, so should be more consistent with the rest of the structure:
```suggestion
grails.plugin.databasemigration:
    changelogLocation: grails-app/migrations
    changelogFileName: changelog-master.xml
    updateOnStart: true
    updateOnStartFileName: changelog-master.xml
```
Also, if you can move it to where the other `grails` properties are, that will also be cleaner.